cli/command/image/history_test.go

```diff
@@ -93,13 +91,15 @@ func TestNewHistoryCommandSuccess(t *testing.T) {
 		},
 	}
 	for _, tc := range testCases {
-		cli := test.NewFakeCli(&fakeClient{imageHistoryFunc: tc.imageHistoryFunc})
-		cmd := NewHistoryCommand(cli)
-		cmd.SetOut(io.Discard)
-		cmd.SetArgs(tc.args)
-		err := cmd.Execute()
-		assert.NilError(t, err)
-		actual := cli.OutBuffer().String()
-		golden.Assert(t, actual, fmt.Sprintf("history-command-success.%s.golden", tc.name))
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
<3 subtests

I think it'd be good to capture the `tc` variable in the loop here, in case we'd ever decide to use `t.Parallel()`;

```suggestion
		tc := tc
		t.Run(tc.name, func(t *testing.T) {
```

Reply:
I'm not sure this is strictly needed with `t.Parallel`, but I also don't think it hurts either.